ostd/src/arch/x86/boot/smp.rs

```diff
@@ -214,7 +214,7 @@ extern "C" {
 unsafe fn wake_up_aps_via_mailbox(num_cpus: u32) {
     use acpi::platform::wakeup_aps;
 
-    use crate::arch::x86::kernel::acpi::AcpiMemoryHandler;
+    use crate::arch::kernel::acpi::AcpiMemoryHandler;
```

Comment:
I believe that code outside the `crate::arch` directory should use `crate::arch` without specifying `crate::arch::x86` or `crate::arch::riscv`.

However, within `crate::arch::x86`, it is acceptable to import other structs using `crate::arch::x86`. For example, in the [standard library](https://github.com/rust-lang/rust/blob/master/library/std/src/os/windows/io/raw.rs#L6), code in `os/windows` will import Windows-specific structs using `crate::os::windows::*`.

Why did you introduce these changes?

Reply:
@StevenJiang1110 `crate::arch::x86` is no longer a valid module or path; there is only `crate::arch`. So even under `arch/x86`, one must use `crate::arch`.